static/app/views/issueDetails/streamline/foldSection.stories.tsx

```diff
@@ -0,0 +1,166 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {IconAdd, IconCopy, IconSubtract} from 'sentry/icons';
+import storyBook from 'sentry/stories/storyBook';
+import {
+  FoldSection,
+  FoldSectionKey,
+} from 'sentry/views/issueDetails/streamline/foldSection';
+
+export default storyBook('FoldSection', story => {
+  story('Usage', () => (
+    <Fragment>
+      <CodeSnippet language="jsx">
+        {`import {FoldSection, FoldSectionKey} from 'sentry/views/issueDetails/section';
+
+<FoldSection title="My Section" sectionKey={FoldSectionKey.MY_SECTION}>
+  <MySectionComponent />
+</FoldSection>`}
+      </CodeSnippet>
+      <p>
+        The <code>SectionKey</code> required in props is used to create a local storage
+        state key to remember the user's previous state for the fold section.
+      </p>
+    </Fragment>
+  ));
+
+  story('Default example', () => {
+    return (
+      <Fragment>
+        <FoldSection title="Default Section" sectionKey={FoldSectionKey.HIGHLIGHTS}>
+          <Lorem />
+        </FoldSection>
+        <CodeSnippet language="jsx">
+          {`<FoldSection title="Default Section" sectionKey={FoldSectionKey.MY_SECTION}>
+  <p>Lorem ipsum...</p>
+</FoldSection>`}
+        </CodeSnippet>
+      </Fragment>
+    );
+  });
+
+  story('Preventing user from collapsing the section', () => {
+    return (
+      <Fragment>
+        <FoldSection
+          title="Prevent Collapse Section"
+          sectionKey={FoldSectionKey.HIGHLIGHTS}
+          preventCollapse
+        >
+          <Lorem />
+        </FoldSection>
+        <CodeSnippet language="jsx">
+          {`<FoldSection title="Prevent Collapse Section" sectionKey={FoldSectionKey.MY_SECTION} preventCollapse>
+  <p>Lorem ipsum...</p>
+</FoldSection>`}
+        </CodeSnippet>
+      </Fragment>
+    );
+  });
+
+  story('Custom headers/titles for the section', () => {
+    return (
+      <Fragment>
+        <FoldSection
+          title={<span style={{color: 'rebeccapurple'}}>Custom Title</span>}
+          sectionKey={FoldSectionKey.HIGHLIGHTS}
+        >
+          <Lorem />
+        </FoldSection>
+        <FoldSection
+          title={
+            <Title>
+              <div>Header with Actions</div>
+              <ButtonBar gap={1}>
+                <Button size="xs" icon={<IconAdd />}>
```

Comment:
might be because it's a test button, but clicking these buttons collapses the section

Reply:
Is this still the case, i wasn't able to repro

https://github.com/user-attachments/assets/4aed6e83-1e77-4773-a3d6-4a29e265b9e4

Reply:
nope - it looks good now!